Sources/PrimerSDK/Classes/User Interface/Components/PrimerResultViewController.swift

```diff
@@ -65,16 +65,8 @@ internal class PrimerResultViewController: PrimerViewController {
                                 value: "Success!",
                                 comment: "Success! - Success Screen Title")
             
-        } else if self.message == nil {
-            self.message = NSLocalizedString("primer-error-screen",
-                              tableName: nil,
-                              bundle: Bundle.primerResources,
-                              value: "Something went wrong, please try again.",
-                              comment: "Something went wrong, please try again. - Error Screen Message")
```

Comment:
I guess this has been removed completely as we now `throw` the errors. Justa asking to check in 💪